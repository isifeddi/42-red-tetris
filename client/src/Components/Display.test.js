import { render } from "@testing-library/react";
import Display from "./Display";

test("Display render test ", () => {
  render(<Display />);
});
