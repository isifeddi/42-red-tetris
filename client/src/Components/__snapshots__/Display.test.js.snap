// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display render test  1`] = `
<div
  className="sc-bdfBwQ hVNBCf"
>
  <div
    className="sc-dlfnbm eTjct"
  />
  <div
    className="sc-hKgILt bOsRbQ"
  />
</div>
`;
